src/flags/general.cpp

```diff
@@ -65,7 +65,10 @@ DEFINE_bool(allow_load_csv, true, "Controls whether LOAD CSV clause is allowed i
 // Storage flags.
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 DEFINE_VALIDATED_uint64(storage_gc_cycle_sec, 30, "Storage garbage collector interval (in seconds).",
-                        FLAG_IN_RANGE(1, 24 * 3600));
+                        FLAG_IN_RANGE(1, 24UL * 3600));
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_VALIDATED_uint64(storage_gc_jemalloc_cycle_sec, 30, "Storage jemalloc garbage collector interval (in seconds)",
+                        FLAG_IN_RANGE(1, 24UL * 3600));
```

Comment:
Why not put this under `storage_ch_cycle_sec`? Assume users don't know anything about allocations. How would they know to tweak this? If we keep this flag, we need to add a very detailed description of what does this do and how to use it properly and in which cases.

How does the `gc_cycle_sec` relate to this flag? E.g. what happens if GC sec is small vs `jemalloc_cycle_sec` is big, and the other way around?

Reply:
I am more than okay to have two separate flags, one for GC for our internal stuff, one for jemalloc. I think users already are impacted by jemalloc via vm.max_map_count and they can read about that already there, so it is one more config that they can set up. Also if jemalloc purge blocks the whole thread, I think it will be too slow to do gc cycles and it might be an issue later on.

Reply:
@antoniofilipovic 
As far as Fico told me purging jemalloc memory is safe and doesn't require any lock, but he will know more for sure

Reply:
Aha, so, on one hand, there is a concern that one scheduler/thread would block the GC, while on the other side there are two flags for the user. Isn't the optimal solution to have one flag `storage_gc_cycle_sec` but two schedulers (both configured with the same flag)?